src/main/java/io/jlu/alfredo/commands/ShowCommand.java

```diff
@@ -19,15 +23,28 @@ public void handleEvent(MessageReceivedEvent event, String parameter) {
         MessageChannel channel = event.getChannel();
 
         try {
-            List<String> workouts = this.jdbi.withHandle(handle ->
-                    handle.createQuery("SELECT Workout FROM Workouts").mapTo(String.class).list());
-
-            StringBuffer workoutsMessage = new StringBuffer();
-            for (int i = 0; i < workouts.size(); i++) {
-                workoutsMessage.append("Recorded Workout: " + workouts.get(i) + "\n");
+            List<Workout> workouts = this.jdbi.withHandle(handle ->
+                    handle.createQuery("SELECT * FROM Workouts")
+                            .map((rs, ctx) -> new Workout(rs.getLong("id"), rs.getLong("time"),
+                                    rs.getString("workout"), rs.getInt("sets"),
+                                    rs.getInt("reps"), rs.getInt("weight")))
+                            .list());
+
+            EmbedBuilder workoutsEmbed = new EmbedBuilder();
+
+            //TODO: This stuff needs to ideally go into some config file somewhere
+            Member umaru = event.getGuild().getMemberById(482570873002262538L);
+            workoutsEmbed.setTitle("Recorded Workouts <:TohruPoint:507804401436459008>");
+            workoutsEmbed.setThumbnail(event.getAuthor().getAvatarUrl());
+            workoutsEmbed.setAuthor(umaru.getNickname(), "", umaru.getUser().getAvatarUrl());
+            workoutsEmbed.setColor(new Color(126, 249, 255));
+
+            if (workouts.size() == 0) {
+                workoutsEmbed.setTitle("Something went wrong");
```

Comment:
Not necessarily an error state, could just be empty state "You have no workouts recorded. Record a workout with..."